deploy/build/contracts/TransferProxy.json

```diff
@@ -0,0 +1,2073 @@
+                              "referencedDeclaration": 13035,
+                              "type": "address",
+                              "value": "to"
+                            },
+                            "id": 13051,
+                            "name": "Identifier",
+                            "src": "746:2:94"
+                          },
+                          {
+                            "attributes": {
+                              "overloadedDeclarations": [
+                                null
+                              ],
+                              "referencedDeclaration": 13037,
+                              "type": "uint256",
+                              "value": "id"
+                            },
+                            "id": 13052,
+                            "name": "Identifier",
+                            "src": "750:2:94"
+                          },
+                          {
+                            "attributes": {
+                              "overloadedDeclarations": [
+                                null
+                              ],
+                              "referencedDeclaration": 13039,
+                              "type": "uint256",
+                              "value": "value"
+                            },
+                            "id": 13053,
+                            "name": "Identifier",
+                            "src": "754:5:94"
+                          },
+                          {
+                            "attributes": {
+                              "overloadedDeclarations": [
+                                null
+                              ],
+                              "referencedDeclaration": 13041,
+                              "type": "bytes calldata",
+                              "value": "data"
+                            },
+                            "id": 13054,
+                            "name": "Identifier",
+                            "src": "761:4:94"
+                          }
+                        ],
+                        "id": 13055,
+                        "name": "FunctionCall",
+                        "src": "717:49:94"
+                      }
+                    ],
+                    "id": 13056,
+                    "name": "ExpressionStatement",
+                    "src": "717:49:94"
+                  }
+                ],
+                "id": 13057,
+                "name": "Block",
+                "src": "707:66:94"
+              }
+            ],
+            "id": 13058,
+            "name": "FunctionDefinition",
+            "src": "542:231:94"
+          }
+        ],
+        "id": 13059,
+        "name": "ContractDefinition",
+        "src": "174:601:94"
+      }
+    ],
+    "id": 13060,
+    "name": "SourceUnit",
+    "src": "33:743:94"
+  },
+  "compiler": {
+    "name": "solc",
+    "version": "0.7.6+commit.7338295f.Emscripten.clang"
+  },
+  "networks": {
```

Comment:
Here addresses should be inserted too for all networks: mainnet, ropsten, rinkeby. Pls, check also other meaningful contracts